File: cpp/bc2547.cpp
#include <iostream>

using namespace std;

int main()
{
	int aMi, aMax, nPe, count;
	
	while(cin >> nPe >> aMi >> aMax)
	{
		count = 0;
		int vec[nPe];
		
		for(int i = 0; i < nPe; i++)
			cin >> vec[i];
		
		for(int i = 0; i < nPe; i++)
			if(vec[i] >= aMi && vec[i] <= aMax) count++;
		
		cout << count << endl;
	}	
}

File: python/bc2547.py
while True:
    count = 0
    try:
        cVal = input().split(" ")

        nPe = int(cVal[0])
        aMi = int(cVal[1])
        aMax = int(cVal[2])

        vec = [int] * nPe

        for c in range(0, nPe):
            vec[c] = int(input())

        for c in range(0, nPe):
            val = vec[c]
            if val >= aMi and val <= aMax:
                count += 1

        print(count)

    except EOFError:
        break
